SU-HSDO/suhumsci
# feat(shs-4993): adjust link color on warbler color scheme

# READY FOR REVIEW

## Summary
Switches the link colors in Warbler color scheme of Traditional theme from tertiary to secondary. 

## Need Review By 8/16

## Urgency
_medium_

## Steps to Test
1. View site in traditional theme
2. Switch color scheme to "Warbler"
3. Note that text links are using #005b6c and hover states are using #616a5d
